R/carehomes.R

```diff
@@ -1070,87 +1246,211 @@ carehomes_parameters_waning <- function(waning_rate) {
 ##'
 ##' @title Carehomes progression parameters
 ##'
-##' @return A list of parameter values
-##'
-##' @param rel_gamma_A Vector of relative rates of gamma_A for each
-##'   strain modelled. If `1` all strains have same rates. Otherwise vector of
-##'   same length as `strain_transmission`, with entries that determines the
-##'   relative scaling of the defaults for each strain.
-##'
-##' @param rel_gamma_P Vector of relative rates of gamma_P for each
-##'   strain modelled. If `1` all strains have same rates. Otherwise vector of
-##'   same length as `strain_transmission`, with entries that determines the
-##'   relative scaling of the defaults for each strain.
+##' @param dt The step size
+##'
+##' @param gamma_E A list with `date` and `value` for changes in gamma_E. If
```

Comment:
Save yourself some pain to update these and describe the format in the details, then just refer to the values.

```r
##' @section Time-varying parameters:
##' Every time varying parameter has the same format, as a list with 
##'   `date` and `value` for changes in the parameter. If
##'   `value` is `NULL` then a default value is used. If `value` is scalar then
##'   `date` can be `NULL` or unspecified. If `value` is a vector then `date`
##'   must be a vector of sircovid dates of the same length as `value` 
##'
##' [....]
##'
##' @param gamma_E Time-varying parameter describing gamma of E (what is this!?). See Details.
```

Reply:
Good idea, thanks, have done this